documentation/user_docs/docs/manual/Advanced_Multifit.rst

```diff
@@ -2,33 +2,73 @@
 Advanced Multifit
 #################
 
-This document describes how to set up an object to work with Multifit specifications
+Parallel fitting
+================
+
+It is possible to use ``multifit`` and its derivatives (``tobyfit``, ``multifit_sqw``, ``multifit_sqw_sqw``) in parallel
```

Comment:
I would have said that tobyfit is a derivative of multifit_sqw - both do the basic fit, while tobyfit convolutes the prediction with the instrument "widths"

Reply:
I would say that they're all derivatives of `multifit` (and its `mfclass`), however I can change the order if you wish. They are all fitting objects inheriting from each other and ultimately `mfclass`.